modules/tinymce/src/themes/silver/test/ts/browser/editor/core/ChoiceControlsTest.ts

```diff
@@ -108,6 +108,18 @@ describe('browser.tinymce.themes.silver.editor.core.ChoiceControlsTest', () => {
           await pAssertOptions(editor, spec.menuSelector, [ '1', '1.1', '1.2', '1.3', '1.4', '1.5', '2' ], Optional.some('1.1'));
           spec.close(editor, 'Line height');
         });
+
+        it(`TINY-7713: ${spec.name} updates if computed line height changes`, async () => {
```

Comment:
This test is failing on Safari, likely due to the same reason as https://github.com/tinymce/tinymce/blob/837146ecbec7f4508258590d039823fa17322f3c/modules/tinymce/src/core/test/ts/browser/commands/LineHeightTest.ts#L28